backend/development.ini.sample

```diff
@@ -430,6 +429,9 @@ script_location = tracim_backend/migration
 ### LOGGERS ###
 [loggers]
 keys = root, tracim, sqlalchemy, alembic, hapic
+# add Tracim_notification logfile
```

Comment:
You should explain here that tracim_notification is used for logging email prepared and send from tracim. this allow to investigate cases where mails are not received by users